app/casa-curator.hs

```diff
@@ -58,25 +59,27 @@ share
   [persistLowerCase|
 LastPushed
   blobId BlobId
+LastDownloaded
+  hackageCabalId HackageCabalId
 SnapshotLoaded
   name Text
   timestamp UTCTime
   Unique SnapshotLoadedNameUnique name
 |]
 
-data CasaPush =
-  CasaPush
-    { _casaPushPantry :: !PantryApp
-    , _casaPushResourceMap :: !ResourceMap
+data PantryStorage =
```

Comment:
Not a big deal, but did you mean to name this `CasaApp` instead? Naming looks a bit inconsistent.

Reply:
Oh, the fields should be `_pantryStorage`, because this type just exists to access the storage in pantry in combination with a resource map.

Reply:
Fixed.